service.autosubs/addon.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="service.autosubs"
+    name="AutoSubs" version="1.1.0" provider-name="amet,schumi2004">
+  <requires>
+    <import addon="xbmc.python" version="3.0.0"/>
+  </requires>
+  <extension point="xbmc.service" library="default.py" />
+  <extension point="xbmc.addon.metadata">
+    <summary lang="en_GB">Auto Search for Subtitles on Play</summary>
+    <description lang="en_GB">Auto Search for Subtitles on Playback start if no subs are found.[CR][CR]Icon credits goes to zNET Computer Solutions http://www.znetcs.pl</description>
+    <platform>all</platform>
+    <language>en_GB</language>
+    <license>GPL</license>
+    <source>https://github.com/amet/service.autosubs</source>
```

Comment:
change upstream location

Reply:
Actually amet is merging my changes to his project. I don't want to take all the credit from him.

Reply:
that's completely fine